rclcpp/test/benchmark/CMakeLists.txt

```diff
@@ -64,6 +64,11 @@ macro(rclcpp_benchmarks)
     benchmark_node
     benchmark_node.cpp
     TEST_OPTIONS ${SKIP_TEST})
+  add_rclcpp_benchmark(benchmark_executor benchmark_executor.cpp
+    AMENT_DEPENDS
+      test_msgs
+    TEST_OPTIONS
+      ${SKIP_TEST})
```

Comment:
Do we actually want to skip this on Connext? I thought the reason behind that was due to performance problems with node shutdown in the timing loop, which I don't see happening in these benchmarks.